docs/sphinx_utils.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
I think it's fine to keep this on our side, however the other sphinx extensions we have are in `gammapy/utils/docs.py`, maybe move to there?

Reply:
Done. 
I did not manage to use exactly the same logic as was previously used. So I added the `setup` in the `conf.py`.

Should we test this code? I suppose we should, but it is not clear how. and what to test.